app/logic/participants.py

```diff
@@ -82,3 +82,16 @@ def unattendedRequiredEvents(program, user):
             return attendedRequiredEventsList
     else:
         return []
+
+
+def getPresentParticipants(event):
+        presentParticipantsDict = {}
+        presentParticipants = (EventParticipant
+            .select()
+            .where(EventParticipant.event==event)
+            .filter(EventParticipant.attended==1))
+
+        for participant in presentParticipants:
+            presentParticipantsDict[participant.user.username] = participant.hoursEarned
```

Comment:
You don't need to change it if you don't want to since the current code is clear, but you could replace everything but the query in this function with a dictionary comprehension: `return {p.user.username: p.hoursEarned for p in presentParticipants}`